bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/transport/MiIoAsyncCommunication.java

```diff
@@ -379,7 +379,7 @@ public void close() {
             }
             senderThread.interrupt();
         } catch (Exception e) {
-            logger.debug("Error while closing ", e.getMessage());
+            logger.debug("Error while closing: {} ", e.getMessage());
```

Comment:
If you catch all `Exception` you are potentially also catching for example `NullPointerException`s and  they tend to miss a message, so you will be printing the message `null` which will make it troublesome to debug, in this case it is always better to log the complete stacktrace.

Reply:
This was easy to replace since the code is so small. Is now `SecurityException`